Dockerfile

```diff
@@ -7,6 +7,8 @@ WORKDIR /app
 ADD . /app
 COPY Pip* /app/
 
+COPY /tmp/git_version /app/git_version
```

Comment:
`/tmp/git_version` is assumed to exist, but nothing is creating this file when building with `make`

Reply:
Its created in `scripts/deploy.sh`, 9 line:
```shell
./git_version_for_docker.sh > "$dest_dir"
```